.eslintrc

```diff
@@ -30,11 +30,14 @@ parser: espree
 env:
   browser: true
   node: true
+  commonjs: false
```

Comment:
It only needs to be turned on if the linter complains about CommonJS syntax being missing (like `modules.exports`). This environments section turns on expected variables and the sort (which could also be manually added to the `globals` section). It's mostly useful for testing where there would be an eslintrc override file in the testing directory that turned on mocha, protractor, etc.